docs/content/guide/security.ngdoc

```diff
@@ -100,7 +100,7 @@ Protection from JSON Hijacking is provided if the server prefixes all JSON reque
 AngularJS will automatically strip the prefix before processing it as JSON.
 For more information please visit {@link $http#json-vulnerability-protection JSON Hijacking Protection}.
 
-Bear in mind that calling `$http.jsonp`, like in [our Yahoo! finance example](https://docs.angularjs.org/guide/concepts#accessing-the-backend),
+Bear in mind that calling `$http.jsonp`, like in [our currency exchange example](https://docs.angularjs.org/guide/concepts#accessing-the-backend),
```

Comment:
Actually, this `line in...` clause is obsolete. We no longer use `jsonp` in that example for quite some time.
Please remove that part.